__tests__/autopilot.test.js

```diff
@@ -1,49 +1,6 @@
-const integration = "autopilot";
-const name = "Autopilot";
 
-const fs = require("fs");
-const path = require("path");
-const version = "v0";
 
-const transformer = require(`../${version}/destinations/${integration}/transform`);
-const inputDataFile = fs.readFileSync(
-  path.resolve(__dirname, `./data/${integration}_input.json`)
-);
-const outputDataFile = fs.readFileSync(
-  path.resolve(__dirname, `./data/${integration}_output.json`)
-);
-const inputData = JSON.parse(inputDataFile);
-const expectedData = JSON.parse(outputDataFile);
+const { executeTransformationTest } = require("./utilities/test-utils");
 
-// Router Test Data
-const inputRouterDataFile = fs.readFileSync(
-  path.resolve(__dirname, `./data/${integration}_router_input.json`)
-);
-const outputRouterDataFile = fs.readFileSync(
-  path.resolve(__dirname, `./data/${integration}_router_output.json`)
-);
-const inputRouterData = JSON.parse(inputRouterDataFile);
-const expectedRouterData = JSON.parse(outputRouterDataFile);
-
-describe(`${name} Tests`, () => {
-  describe("Processor Tests", () => {
-    inputData.forEach((input, index) => {
-      it(`${name} - payload: ${index}`, async () => {
-        try {
-          const output = await transformer.process(input);
-          expect(output).toEqual(expectedData[index]);
-        } catch (error) {
-          expect(error.message).toEqual(expectedData[index].error);
-        }
-      });
-    });
-  });
-  
-  describe("Router Tests", () => {
-    it("Payload", async () => {
-      const routerOutput = await transformer.processRouterDest(inputRouterData);
-      expect(routerOutput).toEqual(expectedRouterData);
-    });
-  });
-  
-});
+executeTransformationTest('autopilot', 'processor')
```

Comment:
There is a difference b/w the display name and the name of integration.
`Facebook Custom Audience` vs `fb_custom_audience`
The support for that is missing here.